modules/osdk-cli-reference-new.adoc

```diff
@@ -22,37 +22,34 @@ The `operator-sdk new` command creates a new Operator application and generates
 |Flag |Description
 
 |`--api-version`
-|CRD API version in the format `<group_name>/<version>`, for example `app.example.com/v1alpha1`. Used with `ansible` or `helm` types.
+|Kubernetes API version in the format `<group_name>/<version>`, for example `app.example.com/v1alpha1`. 
 
 |`--crd-version`
-|CRD version to generate, for example `v1`. Default: `v1beta1`.
+|CRD version to generate. Default: `v1`.
 
 |`--generate-playbook`
 |Generate an Ansible playbook skeleton. Used with `ansible` type.
 
-|`--header-file <string>`
-|Path to file containing headers for generated Go files. Copied to `hack/boilerplate.go.txt`.
-
 |`--helm-chart <string>`
 |Initialize Helm Operator with existing Helm chart: `<url>`, `<repo>/<name>`, or local path.
 
 |`--helm-chart-repo <string>`
 |Chart repository URL for the requested Helm chart.
 
 |`--helm-chart-version <string>`
-|Specific version of the Helm chart. Default: latest version.
+|Specific version of the Helm chart. Default: latest version. Used only with `helm` type.
```

Comment:
Maybe "Used only with the `helm` type.  Default: latest version."

Regarding the addition of that sentence, the argument name includes `helm`...is it really necessary?  Totally up to you.

Reply:
Yeah. That's the restriction mentioned.